tests/fixtures/excerpts_CRLF/index.liquid

```diff
@@ -1,5 +1,7 @@
 ---
 layout: default.liquid
+pagination:
```

Comment:
Why is this test modified?  I would assume the goal is that no existing test should need changes because our goal is that this should go unnoticed by users.

Reply:
Because the default `per_page` value is 10 and we have 12 posts in this test. So 2 posts were paginated away and the test failed.